docs/sources/panels/value-mappings.md

```diff
@@ -3,14 +3,136 @@ title = "Value mappings"
+![Value mappings example](/static/img/docs/value-mappings/value-mappings-example-8-0.png)
 
-Values mapped via value mappings will skip the unit formatting. This means that a text value mapped to a numerical value will not be formatted using the configured unit.
\ No newline at end of file
+If value mappings are present in a panel, then Grafana displays a summary in the side pane of the panel editor.
+
+## Value mapping examples
+
+Value mappings are displayed differently in different visualizations.
+
+### Time series example
+
+Here's an example showing a Time series visualization with value mappings.
+
+![Value mappings time series example](/static/img/docs/value-mappings/value-mappings-summary-example-8-0.png)
+
+### Stat example
+
+Here's an example showing a Stat visualization with value mappings.
+
+![Value mappings stat example](/static/img/docs/value-mappings/value-mappings-stat-example-8-0.png)
+
+### Bar gauge example
+
+Here's an example showing a Bar gauge visualization with value mappings.
+
+![Value mappings bar gauge example](/static/img/docs/value-mappings/value-mappings-bar-gauge-example-8-0.png)
+
+### Table example
+
+Here's an example showing a Table visualization with value mappings.
+
+![Value mappings table example](/static/img/docs/value-mappings/value-mappings-table-example-8-0.png)
+
+## Value mappings fields
+
+This section describes options in the Value mappings user interface.
+
+> **Note:** Display text and Color are both optional. If you want to only assign colors to the text values, then you can leave the display text empty and the original value is displayed.
+
+### Condition
+
+This column lists the type of condition a value mapping is triggered by and the values.
+
+You can map values to three different conditions:
+
+- **Value** maps text values to a color or different display text. For example, if a value is `10`, I want Grafana to display **Perfection!** rather than the number.
+- **Range** maps numerical ranges to a display text and color. For example, if a value is within a certain range, I want Grafana to display **Low** or **High** rather than the number.
+- **Special** maps special values like `Null`, `NaN` (not a number), and boolean values like `true` and `false` to a display text and color. For example, if Grafana encounters a `null`, I want Grafana to display **N/A**.
+
+You can also use the dots on the left as a "handle" to drag and reorder value mappings in the list.
+
+### Display text
+
+The _display text_ is what Grafana displays instead of a number when the listed condition is met.
+
+You can enter any Ascii character or emoji in this field.
+
+### Color
+
+You can select a color to for Grafana to display the value mapping text in.
+
+- **Set color -** Click **Set color** to see a range of recommended colors. Click **Custom** to choose your own color.
+- **Text color -** Select this option to make the displayed text the selected color.
+- **Transparent -** TORKEL, what does this do?
```

Comment:
> - **Text color -** Select this option to make the displayed text the selected color.

This is not correct, the color option does not necessarily affect the display text color, but backgrounds (depends on visualization used). It can affect display text (for example in stat panels, or table with cell display mode set `Color text`). This option sets the selected color be the curren't themes primary text color (ie white in dark theme black in light theme).

I would remove these, they are part of the color picker, not sure they should be part of the value mappings docs. Transparent is if you want a color fully transparent (Shows panel background), mainly useful in thresholds for graphs.

Reply:
Really? I got color changes, at least sometimes. 

Because they are in the UI, we should address them in the documentation. Can you suggest better definitions?